lib/logstash/outputs/scalyr.rb

```diff
@@ -646,6 +648,9 @@ def build_multi_event_request_array(logstash_events)
         if add_log
           log_json = logs[log_identifier].to_json
         end
+        @stats_lock.synchronize do
+          @multi_receive_statistics[:total_successful_json_conversions] += 1
```

Comment:
Sorry, I misread the code. I think you can actually remove this metric since it just adds unnecessary overhead since we need to call it for every single event.

``total_bignum_json_errors`` should already provide the info we need (how many times we needed to call that function), but I was bit confused by the metric name. 

Perhaps ``total_java_class_cast_errors`` would be a better name since  I also saw some of those exceptions which weren't related to the bignum issue (but sadly those exceptions carried no context / traceback to see where the root cause lies).